__tests__/app.test.js

```diff
@@ -118,10 +119,58 @@ describe("API's", () => {
         .expect(200)
         .then(({ body }) => {
           const { articles } = body;
+          expect(articles.length).toBe(13);
           articles.forEach((article) => {
             expect(article).not.toHaveProperty("body");
           });
         });
     });
   });
+  describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: Sould return an array of comments for an article_id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(typeof comment.comment_id).toBe("number");
+            expect(typeof comment.votes).toBe("number");
+            expect(new Date(comment.created_at)).toBeInstanceOf(Date);
+            expect(typeof comment.author).toBe("string");
+            expect(typeof comment.body).toBe("string");
+            expect(typeof comment.article_id).toBe("number");
+          });
+        });
+    });
+    test("GET 200: Sould return an array of comments for an article_id with most recent comments first", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments.length).toBe(11);
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+        });
+    });
```

Comment:
We need one more 200 test, to check that an existing article with 0 comments returns a 200 and an empty array 🔍